common/middleware/mandate-state-enforcer/mandate-state-enforcer.js

```diff
@@ -11,37 +11,31 @@ const pageToValidMandateStateMap = {
   'cancel': ['AWAITING_DIRECT_DEBIT_DETAILS']
 }
 
+const inProgressMessage = {
+  heading: 'Your Direct Debit mandate is being processed',
+  message: 'Check your confirmation email for details of your mandate.',
+  includeReturnUrl: false
+}
+
 const mandateStateToMessageMap = {
-  USER_CANCEL_NOT_ELIGIBLE: {
-    heading: 'You have cancelled the Direct Debit mandate setup',
-    message: 'Your mandate has not been set up.',
-    includeReturnUrl: true
-  },
-  SUBMITTED: {
-    heading: 'Your Direct Debit mandate is being processed',
-    message: 'Check your confirmation email for details of your mandate.',
-    includeReturnUrl: false
-  },
-  PENDING: {
-    heading: 'Your Direct Debit mandate is being processed',
-    message: 'Check your confirmation email for details of your mandate.',
-    includeReturnUrl: false
-  },
+  SUBMITTED_TO_PROVIDER: inProgressMessage,
+  SUBMITTED_TO_BANK: inProgressMessage,
+  ACTIVE: inProgressMessage,
```

Comment:
Should other mandate states past active also show this? It's probably not so important as a user shouldn't be looking at this page again. I.e. CANCELLED, EXPIRED

Reply:
hmm I know we considered it, but as you say thought it very unlikely that a user would come back to this page that far into the lifecycle of a mandate; the downside would be having to list all of our other internal states held in DDConnector in this map (or change the implementation) which makes things more tightly coupled than I think is necessary. So on balance I don't think we should. Also, I'm not sure that message works for the other states, if its cancelled I think saying 'Your Direct Debit mandate is being processed` is a bit misleading.

Reply:
i think that's fair enough, we wouldn't expect these statuses to occur until way down the line anyway